package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "eslint-plugin-blackbird",
+  "version": "0.0.1",
+  "description": "Blackbird ESLint rules",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Peter Stuart",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "babel src -d lib",
+    "test": "mocha lib/tests --recursive",
+    "lint": "eslint --cache \"src/**/**.js\""
+  },
+  "dependencies": {
+    "requireindex": "~1.1.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.7.0",
+    "@babel/core": "^7.7.2",
+    "@babel/node": "^7.7.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
+    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
+    "@babel/preset-env": "^7.7.1",
+    "babel-eslint": "^10.0.3",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-register": "^6.26.0",
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-known-imports": "^0.0.14",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-scope": "^5.0.0",
+    "mocha": "^3.1.2",
+    "prettier": "^1.19.1"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "license": "ISC"
```

Comment:
What is this license?

Reply:
No idea. That came with the generate boilerplate and I didn't notice it. I'll change it.